components/ILIAS/UI/tests/Base.php

```diff
@@ -308,6 +308,11 @@ public function getCanonicalName(): string
  * Provides common functionality for UI tests.
  */
 abstract class ILIAS_UI_TestBase extends TestCase
+{
+    use BaseUITestTrait;
+}
+
+trait BaseUITestTrait
```

Comment:
Hi @matheuszych,

could you explain why you made this change?

Also, please talk to @mjansenDatabay regarding whom to include for approval of code changes. I was a little irritated, because AFAIR this never passed the desk of the UI coordinators.

Thanks!

Reply:
Hello @klees,

I did that so I could use the functionality provided in another Unit Test (Test/ObjTestScoreSettingsTest.php). In particular, but not only, I am referring to the method "getDefaultRenderer."

Sorry for not informing any of the UI coordinators about these changes. It is such a small change that it must have gone unnoticed among everything else in this PR.

Best Regards,
Matheus Zych

Reply:
Hi @matheuszych,

this is not good, you can't just introduce public interfaces in other peoples code. If I hadn't spotted this and asked, I might come across the code one day and decide to refactor it. Then your unit tests will break and it is on me to fix them.

Kind regards!

Reply:
@mbecker-databay / @kergomard  / @dsstrassner Could you look into this and maybe help @matheuszych to solve the issue?

Reply:
I am unable to provide @matheuszych help in this case, but I kindly ask @mbecker-databay or @mjansen to help him, to solve this in a quick manner. 
I also ask my colleagues to request reviews from the respective authorities from other components, if we want to change code there, before we merge PRs.